tests/test_reader.py

```
@@ -37,9 +38,35 @@ def test_trans_descr(self):
+        """R12 (данные) должен возвращаться сырым значением 1С вместе с внешними {}.
+        Регэксп sel_re (группа 13) срезает обрамляющие скобки, а ЗначениеИзСтрокиВнутр
+        в обработке epf/Nikita требует полное значение вида {"S","..."}."""
+        # Запись собрана строго по sel_re (см. tests/test_parser_regex.py).
+        record = (
+            ",\r\n{20231201120000,N,\r\n"
+            "{0,0},1,2,3,4,5,I,\"Test Comment\",1,\r\n"
+            "{\"S\",\"Data\"},\"Presentation\",6,7,8,9,1,\r\n"
+            "{0}\r\n}"
+        )
+        raw = record.encode("utf-8")
+        tmp = tempfile.NamedTemporaryFile(delete=False, suffix=".lgp")
```

Ruff (0.15.18)

[warning] Use a context manager for opening files

(SIM115)